### ISSUE TITLE
Make it possible to have a "local mode" that won't attempt to connect to discord and broadcast messages about new events/raffles/etc.

### ISSUE THREAD

**Issue by chadjoan** [Kickback-Kingdom/kickback-kingdom-website#21]
This is important for being able to do development and testing on local dev machines without disturbing the public site+community with false alarms and erroneous raffle ticket winners (sorry Hans!).

**Comment by chadjoan** [comment]
This optionally depends on issue #3: `Introduce a new config file for various non-security settings.`
(It'd be nice to have this mode set in a config file, so that machine role can just be "set-and-forgot" in the config file.)
